TestKickStart.bat

```diff
@@ -0,0 +1,8 @@
+
+@ECHO ON
+
+ECHO Python 2.7 Tests ===============================
+py -2.7 TestRunner.py
```

Comment:
OK, glad I asked since I misunderstood what was going on here, I didn't know that "py" was actually a custom "Python Launcher for Windows" that has those options. I also didn't know that Python 3.3+ actually adds that launcher to your path so there is no longer need to add python to your PATH (if you use the "py" command that is).  

So just something to be aware of unless the end user has Pro &/| Python 3.3+ it may require an extra configuration step to add this to the path.

But anyway still seems like a good solution to me, with minimal config required.